hack/bin/run-local.sh

```diff
@@ -21,9 +29,20 @@ function setEnvFromSecrets {
 
   for specenv in $(kubectl get deployment -n "$NAMESPACE" observability-operator -ojson | jq -c -M '.spec.template.spec.containers[0].env[]') ; do
     envname=$(echo "$specenv" | jq -r '.name')
-    secretname=$(echo "$specenv" | jq -r '.valueFrom.secretKeyRef.name')
-    secretkey=$(echo "$specenv" | jq -r '.valueFrom.secretKeyRef.key')
-    envvalue=$(kubectl get secret -n "$NAMESPACE" "$secretname" -ojson | jq -c -M -r '.data["'"$secretkey"'"]' | base64 -d)
+    # Let's look for a direct value
+    envvalue=$(echo "$specenv" | jq -r '.value')
+
+    # If the value is null, we check if it is a secret reference
+    if [[ "$envvalue" == "null" ]]; then
```

Comment:
Pro-tip: you can use the `strings` jq filter to avoid `null` values and only return value being a string.

```suggestion
    envvalue=$(echo "$specenv" | jq -r '.value | strings')

    # If the value is null, we check if it is a secret reference
    if [[ -z "$envvalue" ]]; then
```